pkgs/servers/x11/xorg/overrides.nix

```diff
@@ -1,109 +1,727 @@
+      license = lib.licenses.lucidaBitmapFonts;
+    };
+  });
+
+  fontbh100dpi = prev.fontbh100dpi.overrideAttrs (attrs: {
+    meta = attrs.meta // {
+      license = lib.licenses.lucidaBitmapFonts;
+    };
+  });
+
+  fontbhlucidatypewriter75dpi = prev.fontbhlucidatypewriter75dpi.overrideAttrs (attrs: {
+    meta = attrs.meta // {
+      # only includes a copyright
+      license = lib.licenses.unfree;
+    };
+  });
+
+  fontbhlucidatypewriter100dpi = prev.fontbhlucidatypewriter100dpi.overrideAttrs (attrs: {
+    meta = attrs.meta // {
+      # only includes a copyright
+      license = lib.licenses.unfree;
+    };
+  });
+
+  fontbhttf = prev.fontbhttf.overrideAttrs (attrs: {
+    meta = attrs.meta // {
+      # Copyright notice permits using and distributing, but changing is explicitly disallowed.
+      # Wikipedia says they are non-free (quoting Debian):
+      # https://en.wikipedia.org/wiki/Luxi_fonts
+      # https://web.archive.org/web/20141006121747/https://packages.debian.org/wheezy/ttf-xfree86-nonfree
+      license = lib.licenses.unfreeRedistributable;
+    };
+  });
+
+  fontbhtype1 = prev.fontbhtype1.overrideAttrs (attrs: {
+    meta = attrs.meta // {
+      # same license as fontbhttf
+      license = lib.licenses.unfreeRedistributable;
+    };
+  });
+
+  fontbitstream100dpi = prev.fontbitstream100dpi.overrideAttrs (attrs: {
+    meta = attrs.meta // {
+      license = with lib.licenses; [
+        free
+        bitstreamCharter
+      ];
+    };
+  });
+
+  fontbitstream75dpi = prev.fontbitstream75dpi.overrideAttrs (attrs: {
+    meta = attrs.meta // {
+      license = with lib.licenses; [
+        free
+        bitstreamCharter
+      ];
+    };
+  });
+
+  fontbitstreamtype1 = prev.fontbitstreamtype1.overrideAttrs (attrs: {
+    nativeBuildInputs = attrs.nativeBuildInputs ++ [ fontforge ];
+
+    postBuild = ''
+      # convert Postscript (Type 1) font to otf
+      for i in $(find -type f -name '*.pfa' -o -name '*.pfb'); do
+          name=$(basename $i | cut -d. -f1)
+          fontforge -lang=ff -c "Open(\"$i\"); Generate(\"$name.otf\")"
+      done
+    '';
+
+    postInstall = ''
+      # install the otf fonts
+      fontDir="$out/lib/X11/fonts/misc/"
+      install -D -m 644 -t "$fontDir" *.otf
+      mkfontscale "$fontDir"
+    '';
+
+    meta = attrs.meta // {
+      license = lib.licenses.bitstreamCharter;
+    };
+  });
+
+  fontbitstreamspeedo = prev.fontbitstreamspeedo.overrideAttrs (attrs: {
+    meta = attrs.meta // {
+      license = lib.licenses.bitstreamCharter;
+    };
   });
 
-  fonttosfnt = super.fonttosfnt.overrideAttrs (attrs: {
+  fontcronyxcyrillic = prev.fontcronyxcyrillic.overrideAttrs (attrs: {
+    meta = attrs.meta // {
+      license = lib.licenses.cronyx;
+    };
+  });
+
+  fontcursormisc = prev.fontcursormisc.overrideAttrs (attrs: {
+    meta = attrs.meta // {
+      # whatever this is supposed to mean:
+      # "These ""glyphs"" are unencumbered"
+      license = lib.licenses.unfree;
```

Comment:
Pretty standard *ye olde style* somewhere‐between‐permissive‐and‐public‐domain notice. Probably not even copyrightable in the country of origin (remember that typefaces and bitmap fonts are considered too functional to copyright in the US).

You should cross‐reference all of these you can with [Fedora’s package specification](https://src.fedoraproject.org/rpms/xorg-x11-fonts/blob/21908b7629fd3c15cf24b8f0fb63a221e0a3d655/f/xorg-x11-fonts.spec); in this case:

```
# SPDX: LicenseRef-Fedora-UltraPermissive
Source16:   https://www.x.org/pub/individual/font/font-cursor-misc-1.0.3.tar.bz2
```